docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -186,6 +186,18 @@ Directory path for seccomp profiles (default: "/var/lib/kubelet/seccomp"). (This
 
 Start the pod after creating it, set to false to only create it.
 
+#### **--wait**, **-w**
```

Comment:
Shouldn't this be a little later in the manpage, if we're doing alpha order? After userns.container?

Reply:
is userns.container a flag? It says options and has a different format than all the flags defined, that is why I put --wait above it. Can move it if it should be under that

Reply:
It's basically including another file at that spot that has the test of the `userns.container` flag

Reply:
yup build complained about it too. Fixed